src/loaders/lottie/tvgLottieBuilder.cpp

```diff
@@ -1308,7 +1308,7 @@ static void _attachFont(LottieComposition* comp, LottieLayer* parent)
             auto len2 = strlen(font->name);
             if (!strncmp(font->name, doc.name, len < len2 ? len : len2)) {
```

Comment:
thanks, could you please review with this?
```
if (len == len2 && !strcmp(font->name, doc.name)) {
```

Reply:
It works!